libkbfs/favorites.go

```diff
@@ -125,15 +211,18 @@ func (f *Favorites) handleReq(req *favReq) (err error) {
 	//  * The user asked us to refresh
 	//  * We haven't fetched it before
 	//  * It's stale
-	if req.refresh || f.cache == nil || time.Now().After(f.cacheExpireTime) {
+	if req.refresh || f.cache == nil || time.Now().After(f.
```

Comment:
Please use `f.config.Clock().Now()` instead of `time.Now()`.  (I missed that in an unreviewed update to an old PR, sorry.)